WcaOnRails/app/views/competitions/show.pdf.erb

```diff
@@ -0,0 +1,46 @@
+<!doctype html>
+<html>
+  <head>
+    <%= wicked_pdf_stylesheet_link_tag "pdf" -%>
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+  </head>
+  <body>
+    <header>
+      <div class="pdf-logo">
+        <%= wicked_pdf_image_tag "wca_logo.svg" %>
+      </div>
+      <div class="pdf-title"><%= @competition.name %></div>
+      <div class="comp-date">
+        <%= wca_date_range(@competition.start_date, @competition.end_date) %>
+      </div>
+      <div class="separator"></div>
+      <div>
+        <%= md @competition.venue %>
+        <p>
+          <%= @competition.venueAddress %>
+        </p>
+        (<%= link_to_google_maps_place "#{@competition.latitude_degrees}, #{@competition.longitude_degrees}", @competition.latitude_degrees, @competition.longitude_degrees %>)
+      </div>
+      <div class="separator"></div>
+      <div><%= @competition.city_and_country %></div>
+    </header>
+    <div class="main_content">
+      <% formats_list = [] %>
```

Comment:
We pass that array to the partial just to append the formats in use, right? This seems unnecessarily tricky, any reason for not computing it here?
```rb
formats_list = @competition.rounds.flat_map(&:formats_used).map(&:id).uniq
```